cycles-ledger/src/storage.rs

```diff
@@ -121,6 +121,12 @@ pub enum Operation {
     Mint {
         to: Account,
         amount: u128,
+        // Custom non-standard fee to record the amount
+        // of cycles "burn" when cycles are deposited, i.e.
+        // the diffence between the cycles deposited and
+        // the cycles minted. Note that this field has
+        // no effect on the balance of the `to` account.
```

Comment:
What do you mean by it has "no effect"? The fee is deducted from the deposited amount, so the `to` account is affected in that it does get less.

Reply:
Balance in this context is the balance on the Ledger. It means that a client processing this block should not use the new field to calculate the balance of the user. This is different form the other `fee` fields, e.g. in `transfer` or `approve`, which are used to calculate the balance of the user.